### DISCUSSION TITLE
Server Problems with the annotation tool

### DISCUSSION THREAD

**Discussion by drviver** [sebastian-hofstaetter/teaching#57] [Advanced IR: Exercise Q&A] [answered]
When I go on the link, it just gives me an Internal Server Error 500.
I tried it in multiple browsers and different internet connections, as well as my smart phone .
![image](https://user-images.githubusercontent.com/11941298/161957033-d6fdc52f-a3d9-4f00-ae4a-a1e0f7a31660.png)

!!Update moved thread to issues

**Comment by sophiaalthammer** [comment]
It is an internal server error, we will get back to you, when we fixed it, sorry!

Best,
Sophia

**Comment by sebastian-hofstaetter** [comment, marked as answer]
Thanks for flagging this, it was a short hiccup; fixed it :)
Best,
Sebastian
